h3.go

```diff
@@ -67,6 +67,59 @@ const (
 	RadsToDegs = 180.0 / math.Pi
 )
 
+var H3ErrorFailed = errors.New("Unknown error")
```

Comment:
In addition to the following lint warnings, can you group them in a var block like
```go
var (
  ErrFailed = ...
  ErrDomain = ...
  ..
)
```

and update the error messages to start lowercase 
https://github.com/golang/go/wiki/CodeReviewComments#error-strings